frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/virtual-hardware-tab/virtual-hardware-tab.scss

```diff
@@ -0,0 +1,29 @@
+.kubevirt-create-vm-modal__virtual-hardware-tab-container {
+  display: flex;
+  flex-direction: column;
+  min-height: 100%;
+
+  .kubevirt-create-vm-modal__virtual-hardware-tab-cd-title {
```

Comment:
we probably shouldn't specify children like this if it is not necessary. I.e. it is already fully identified by the class itself